tools/sst_dump_tool.cc

```diff
@@ -588,6 +593,17 @@ int SSTDumpTool::Run(int argc, char** argv) {
       }
     }
 
+    if (command == "verify") {
```

Comment:
That is different. check and verify_checksum will do that every time when open a new bbtr in compaction, reads, etc. but verify will only do that once as a command.